epics/cache.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Provides a cache outside of Redux state that can optimistically update state
+ * before an asynchronous API call returns
+ *
+ * @module CacheMiddleware
+ */
+import Rx from 'rxjs';
```

Comment:
That's my ambivalence on import style leaking through. I don't have a strong preference for either style, or a particularly strong need to be consistent, _but_ I'm pretty sure there are some ways of updating these imports so that you don't have to import the whole library, which can make the bundle smaller - I'll open a Jira ticket to review the imports and we can hopefully get everything nice and tidy then